internal/datanode/data_node.go

```diff
@@ -962,11 +964,11 @@ func (node *DataNode) Import(ctx context.Context, req *datapb.ImportTaskRequest)
 		AutoIds:    make([]int64, 0),
 		RowCount:   0,
 	}
-	// func to report import state to rootcoord
+	// func to report import state to RootCoord.
 	reportFunc := func(res *rootcoordpb.ImportResult) error {
-		status, err := node.rootCoord.ReportImport(ctx, res)
+		status, err := node.rootCoord.ReportImport(node.ctx, res)
```

Comment:
We shall not use node.ctx for request context. Since grpc deadline and cancel will be missed.

Reply:
Done.